MaterialsLib/CMakeLists.txt

```diff
@@ -4,3 +4,7 @@ GET_SOURCE_FILES(SOURCES_ADSORPTION Adsorption)
 set(SOURCES ${SOURCES} ${SOURCES_ADSORPTION})
 
 add_library(MaterialsLib ${SOURCES} )
+target_link_libraries(MaterialsLib
+    logog
+    BaseLib
```

Comment:
`BaseLib` is linked with `logog` already. So there is no need to link with `logog`.